examples/vnet/doca_flow_chain.c

```diff
@@ -82,8 +100,10 @@ build_flow_isolate(struct doca_flow_port *port,
         flow.attr.priority = 0;
     } else {
         ret = doca_dpdk_build_modify_actions(&pipe_cfg, &flow);
-        if (ret)
+        if (ret) {
             return NULL;
+        }
+        build_jump_action(&pipe_cfg, &flow);
```

Comment:
if isolate than you always jump to 1, but better set as a global definition in priv.h